lesson_01/TommyTran/README.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+
+<link href="style.css" rel="stylesheet"/>
+
+
+<body>
+ <main>
+    <div class="md-heading">
+ <h1> Tommy Tran README</h1>
+    </div>
+    <div class="line"> </div>
+    <div class="md-heading">
+<h2>About Me</h2>
+    </div>
+    <div class="line"> </div>
+<p>Originally born and raised in Springfield, Virginia I started my life in the DMV area.
+ After moving to Delaware I eventually decided to go to a Vocational Tech School where I majored in carpentry.
+I worked on woodworking projects including cabinets, framing, and roofing. I continued my studies in carpentry
+for an additional 2 years before deciding on going to Delaware tech for College. I studied accounting for a time
+ and received my associates degree. Afterwards I entered an adult learning  program where I learned HVAC.
+I worked in this industry for the following 2 years before finding an interest in coding.
+</p>
+    <div class="md-heading">
+<h3>Learning Style</h3>
+    </div>
+
+<p>I learn best in a social setting as I can bounce ideas across other people and receive feedback on my actions immediately.
+ I also like to have a hands on approach on subjects and prefer getting my hands dirty with experiments and role-playing scenarios.
+When I'm alone i like to reflect on the work that I have done and repeat challenge exercises to build up the mental repetition in my head.
+I struggle to figure out how to tackle problems in the beginning but feel that it gets easier once I have built once I have a solid foundation
+ to process my ideas.
+</p>
+    <div class="md-heading">
+<h3>Likes </h3>
+    </div>
+ <ol>
+    <li>Music / Concerts</li>
+    <li>Art / Fashion</li>
+    <li>Traveling</li> 
+ </ol>
+    <div class="md-heading">
+<h3> Dislikes </h3>
+    </div>
+<ol>
+    <li> Seafood</li>
+    <li> Bugs</li>
+    <li>Working on Sundays</li>
+</ol>
+    <div class="md-heading">
+<h3>Goals</h3>
+    </div>
+<ol>
+<li>More music festivals and concerts and meet new people.</li>
+<li>Travel to Japan and go to as many different bars and restaurant.</li> 
+ <li>Be able to work remote anywhere in the world.</li>
+</ol>
+    <div class="md-heading">
+<h3>My Favorite Past Time</h3>
+    </div>
+<img src= "Images/pancakes.jpg" style="width:500px;height:500px;" alt="A hearty breakfast meal"  style="width:500px; height:500px;"></img>
```

Comment:
Picture is not showing, please fix.

Reply:
updated the image line and added changes based off of other comments

Reply:
nested my meta tags and link tag inside a <head> in order to properly load my style.css